<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InsertReferenceDialog</class>
 <widget class="QDialog" name="InsertReferenceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>506</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Insert Reference</string>
  </property>
  <layout class="QGridLayout">
   <item row="2" column="0">
    <widget class="QPushButton" name="helpButton_">
     <property name="text">
      <string>&amp;Help</string>
     </property>
     <property name="shortcut">
      <string>Alt+H</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="5">
    <layout class="QHBoxLayout">
     <item>
      <widget class="QLabel" name="addressLabel_">
       <property name="text">
        <string>URL:</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="addressEdit_">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="browseButton_">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>&amp;WebDAV</string>
       </property>
       <property name="shortcut">
        <string>Alt+W</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="insertButton_">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>&amp;Insert</string>
     </property>
     <property name="shortcut">
      <string>Alt+I</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="insertAndCloseButton_">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Insert &amp;&amp; Cl&amp;ose</string>
     </property>
     <property name="shortcut">
      <string>Alt+O</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QPushButton" name="cancelButton_">
     <property name="text">
      <string>&amp;Close</string>
     </property>
     <property name="shortcut">
      <string>Alt+C</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <spacer name="spacer5">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>220</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="5">
    <widget class="QSplitter" name="splitter2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="handleWidth">
      <number>1</number>
     </property>
     <widget class="QWidget" name="layout3">
      <layout class="QVBoxLayout" name="layout3_2">
       <property name="spacing">
        <number>2</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTreeWidget" name="treeView_">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <column>
          <property name="text">
           <string>Files</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="extComboBox_">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <item>
          <property name="text">
           <string>*.xml;*.dita</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>*.*</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layout6">
      <layout class="QVBoxLayout" name="layout6_2">
       <property name="spacing">
        <number>2</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTreeWidget" name="listView_">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>2</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>400</height>
          </size>
         </property>
         <column>
          <property name="text">
           <string>Select Target</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Type</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="filterComboBox_">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>2</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <item>
          <property name="text">
           <string>All Targets</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Topic</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Step</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>List Item</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Example</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Section</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Figure</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Table</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <tabstops>
  <tabstop>treeView_</tabstop>
  <tabstop>listView_</tabstop>
  <tabstop>helpButton_</tabstop>
  <tabstop>insertButton_</tabstop>
  <tabstop>cancelButton_</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton_</sender>
   <signal>clicked()</signal>
   <receiver>InsertReferenceDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>insertButton_</sender>
   <signal>clicked()</signal>
   <receiver>InsertReferenceDialog</receiver>
   <slot>insertLink()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>extComboBox_</sender>
   <signal>activated(int)</signal>
   <receiver>InsertReferenceDialog</receiver>
   <slot>extensionChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>filterComboBox_</sender>
   <signal>activated(int)</signal>
   <receiver>InsertReferenceDialog</receiver>
   <slot>filterChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>insertAndCloseButton_</sender>
   <signal>clicked()</signal>
   <receiver>InsertReferenceDialog</receiver>
   <slot>insertLinkAndClose()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>browseButton_</sender>
   <signal>clicked()</signal>
   <receiver>InsertReferenceDialog</receiver>
   <slot>browse()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listView_</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>InsertReferenceDialog</receiver>
   <slot>selectionChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>445</x>
     <y>242</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listView_</sender>
   <signal>itemDoubleClicked(QTreeWidgetItem*,int)</signal>
   <receiver>InsertReferenceDialog</receiver>
   <slot>insertLink()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>445</x>
     <y>242</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
